### PR TITLE
Doc: Standardize quickstart tutorials

### PR DESCRIPTION
**Pull request by jaidevd**
Equivalent of https://github.com/gramener/gramex/pull/177

### CODE REVIEW

File: tutorials/g1-dropdown/output/index.html [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+  }
+</script>
+
+{% else %}
+
+<body>
+  <div class="text-center h2">SuperStore Dataset</div>
+  <div class="formhandler" data-src="../../../data?_c=-Order ID&_c=-Order Date&_c=-Ship Date"></div>
+  <div id="chart"></div>
+  <div id="container1"></div>
```
Comment (pratapvardhan) [inline tutorials/g1-dropdown/output/index.html]: @jaidevd -- would it be better to move dropdown above chart / table? It's not visible in default viewport, keeping it on top makes it quick to spot.

File: tutorials/gramex.yaml [hunk 2]
```diff
@@ -16,3 +16,60 @@ url:
...
+    kwargs:
+      path: $YAMLPATH/dashboards/output/index.html
+      transform:
+        "index.html":
+          function: template
+          args: =content
+          kwargs:
+            handler: =handler
+      Content-Type: text/html
+      Cache-Control: no-store
```
Comment (pratapvardhan) [inline tutorials/gramex.yaml]: @jaidevd -- `index.html.source` endpoints are missing are dropdown and dashboards. When you add them, you could consider making three of them generic in one endpoint? `/$YAMLURL/{dashboards|dropdown|other}/output/([0-9]+)/index.html`

File: tutorials/quickstart/README.md [hunk 3]
```diff
@@ -74,70 +86,151 @@ INFO    22-Apr 13:34:26 __init__ 9988 <Ctrl-B> opens the browser. <Ctrl-D> start
...
+    "City":"Henderson","State":"Kentucky","Region":"South","Category":"Furniture",
+    "Sub-Category":"Chairs","Sales":731.94,"Quantity":3,"Discount":0.0,"Profit":219.582
+  },
+  // etc
+]
+```
 
-<script type="text/html" class="step1dataophtml" src="snippets/example-output.html"></script>
+You could also visit
+[http://localhost:9988/tutorials/data?_limit=10&_format=html](http://localhost:9988/tutorials/data?_limit=10&_format=html)
```
Comment (pratapvardhan) [inline tutorials/quickstart/README.md]: Use relative URL maybe? If we want this to accessible from https://learn.gramener.com/guide/

Comment (jaidevd) [reply tutorials/quickstart/README.md]: The localhost URLs are kept like this on purpose, we want the user to actually see things on their own machines.

**Comment by jaidevd** [comment]
@pratapvardhan Done.

**Comment by pratapvardhan** [comment]
Made few tiny changes to yaml config.

**Pull request merged**